src/popover.ts

```diff
@@ -149,6 +163,90 @@ export function apply() {
     },
   });
 
+  const popoverTargetAttributesSupportedElementsSelector =
+    ':is(button, input[type="button"], input[type="submit"], input[type="image"], input[type="reset"])';
+
+  const definePopoverTargetElementProperty = (name: string) => {
+    const invokersMap = new WeakMap<Element, Element>();
+    Object.defineProperty(HTMLElement.prototype, `${name}Element`, {
```

Comment:
I think we probably want to define these on `HTMLButtonElement.prototype` and `HTMLInputElement.prototype`.